lib/etcher.js

```diff
@@ -19,12 +19,47 @@
 const electron = require('electron');
 const path = require('path');
 const elevate = require('./elevate');
+const fs = require('fs');
+const args = require('minimist')(process.argv.slice(2));
 let mainWindow = null;
 
 electron.app.on('window-all-closed', electron.app.quit);
 
 electron.app.on('ready', function() {
 
+  if (args.h || args.help) {
+    console.log('Etcher - A better way to flash OS images to SD cards');
+    console.log('http://etcher.io\n');
+    console.log('Usage:');
+    console.log('   ' + process.argv[0] + ' [options] [image]\n');
+    console.log('Options:');
+    console.log('   -h | --help     This information');
+    console.log('\n');
+    console.log('Supported image formats:');
+    console.log('   .img .iso .zip');
+    console.log('\n');
+    return process.exit(0);
+  }
+
+  if (args._.length > 1) {
+    console.log('Only one image can be specified.');
+    console.log('(Run "' + process.argv[0] + '" -h for more information.)');
+    return process.exit(0);
+  }
+
+  if (args._.length === 1) {
+
+    // Check for image file existence and pass into the writer
+    var stat;
+    try {
+      stat = fs.statSync(args._[0]);
```

Comment:
`fs.access()` is the recommended way to perform these kind of checks now that `fs.exists()` has been deprecated (https://github.com/nodejs/node/issues/1592). Alternatively, I'd be happy to include a new dependency that provides an "existence" check functionality (potentially around `fs.access()`) to more clearly communicate the intention.

For example, having a `fileExists(file)` call reads much nicer than a `stat` or `fs.access()`, for which you have to understand that the real reason you're calling that is to check that the file exists.